e2e/__tests__/globals.test.js

```diff
@@ -20,7 +20,7 @@ const TEST_DIR = path.resolve(DIR, '__tests__');
 
 function cleanStderr(stderr) {
   const {rest} = extractSummary(stderr);
-  return rest.replace(/.*(jest-jasmine2|jest-circus).*\n/g, '');
+  return rest.replace(/.*(jest-jasmine2).*\n/g, '');
```

Comment:
no need to stirp out circus stuff as we always create an error without any circus traces in it. Can't be bothered doing the same fix for jasmine, though